### PR TITLE
Haleta

### PR DESCRIPTION
**Pull request by Ralph-Lee-UK**
Table of melodic variations for the Hallelujah አንቀጽ፡ ሃሌታ፡ for three works as sub part

### CODE REVIEW

File: new/LIT1990Mawase.xml [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+               <label>Antiphon of ʾAbbā Libānos</label>
+            </div>
+            <div type="textpart" corresp="LIT4498Pilgrims">
+               <label>Antiphon of Pilgrims (<foreign xml:lang="gez">ዘፈላስያን፡</foreign>)</label>
+            </div>
+            <div type="textpart" corresp="LIT4499Sunday">
+               <label>Antiphon of Sunday  (<foreign xml:lang="gez">ዘሰንበተ፡ ክርስቲያን፡</foreign>)</label>
+            </div>
+            <div type="textpart" xml:id="Haleta">
+              <label>Table of melodic variations for the Hallelujah <foreign xml:lang="gez">አንቀጽ፡ ሃሌታ፡</foreign></label>
```
Comment (DenisNosnitsin1970) [inline new/LIT1990Mawase.xml:310]: As far as I can see, the spelling is አንቀጸ፡ ሃሌታ (status constructus)

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT1990Mawase.xml (hunk 1 above)
Comment (Ralph-Lee-UK) [reply new/LIT1990Mawase.xml:310]: You are quite correct - in the particular case I was looking at there is a spelling error - but it is indeed an error! I'll correct this in all three cases

**Comment by Ralph-Lee-UK** [comment]
@DenisNosnitsin1970 looking at the EMIP records this also occurs with Deggwa and Soma Deggwa ms. Should we add it there as well?

**Comment by DenisNosnitsin1970** [comment]
I would say, yes.

**Pull request merged**